invisible_cities/cities/components.py

```diff
@@ -493,6 +494,24 @@ def hits_and_kdst_from_files(paths: List[str]) -> Iterator[Dict[str,Union[HitCol
                            timestamp = timestamp)
 
 
+def load_MC(files_in : List[str]) -> Generator:
```

Comment:
As @andLaing  pointed out the name is too generic, and should probably be something of `MC_hits_from_files` or similar

Reply:
Totally agree. Also, we must add the hits `label` in the source. I will push this change right now.